h/services/group.py

```diff
@@ -73,6 +73,32 @@ def create(self, name, authority, userid, description=None, type_='private'):
 
         return group
 
+    def create_open_group(self, name, authority, userid, hostnames, description=None):
```

Comment:
I've put `create_open_group()` in a separate method from the generic `create()` method above. This allows me to avoid needing to use the awkward `GROUP_ACCESS_FLAGS` thing above, and avoid a `type_` param, keeps just the logic for creating open groups together in one place.

This is certainly not what we want in the long run because there's some duplication between `create()` and `create_open_group()`, and because code is still using `create()` to create eLife-style groups which are actually just third-party open groups so should be created using `create_open_group()` (and then `create()` only needs to be able to create private groups, at which point `GROUP_ACCESS_FLAGS` and `type_` are no longer needed). I'm holding back on any refactoring of `GroupService` though so not to step on @lyzadanger 's toes.

In the future what I think we might want is an `OpenGroupService()` with a `create()` and a separate `PrivateGroupService()` with a different `create()` and with `member_join()` and `member_leave()`.